### PR TITLE
Feature/Setup 'Hello world' project as one app with Webpack

### PR DESCRIPTION
**Pull request by RileyManda**
I have completed the task of creating Seting up a 'Hello world' project as one app with Webpack with react and postgres according to the stipulated [Project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/rails_react_webpack.md#project-requirements) 🌱 

- Created a new Rails app called 'hello-rails-react'.
- Created a babel.config.js in the root folder of the project 
- Updated webpack.config.js to include the babel loader 
- Made sure that the project has a Postgres database set up.
- Create a table for storing messages and created 5 different greetings.
- Create an API endpoint that selects a random greeting from the table (Using a controller with an action and Rails route).
- Created a static view that is set as the root of the app.
- Created the App component with react-router and render it in the static view.:
- Created the Greeting component thats displaying a greeting and Set it up as a route in the App component.
- Used redux toolkit to create a store, an action, and a reducer that connects to the API endpoint to get the random greeting.
- Displayed the random greeting in the Greeting component.:as json.
  http://localhost:3000/greetings/random_greeting

Please kindly review my implementation.

Thank you 🙏🏻

### CODE REVIEW

File: app/javascript/redux/store.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import messagesReducer from './reducers/messages';
+
+const store = configureStore({
+    reducer: {
+        messages: messagesReducer,
+    },
```
Comment (TedLivist) [inline app/javascript/redux/store.js:7]: - [ ] I cannot seem to understand which files you were trying to import here as there is neither a `reducers` folder nor a messages file in your Javascript sub-directory of the project. I believe you are trying to import the greetingsReducer, so line 2 should be this:

  ```
  import greetingsReducer from './greetingsSlice';
  ```
  
- [ ] Then you need to configure the store correctly so line 6 should be something like this:

  ```
  greetings: greetingsReducer,
  ```

  If you follow these instructions your app will be correctly set up.

- [ ] Next your app does not work at all. Nothing is being fetched and displayed on the browser. Kindly figure out why and verify your project works before submitting for another review.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @RileyManda,

Good job so far!
✔️ Greeting component has a route
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by RileyManda** [comment]
I have completed the changes requested on this milestone.

## Implementations

Please kindly review my implementation.

Thank you 🙏🏻

**Review by uwadonat — APPROVED** [review]
## Hi @RileyMandam,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Good project structure ✔️
- Detailed commit message ✔️
- No linter errors ✔️
- Professional readme file ✔️

approval

 
# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**